internal/pkg/model/transformation.go

```diff
@@ -7,8 +7,16 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/strhelper"
 )
 
+type UsedSharedCodeRows []ConfigRowKey
+
+type LinkToSharedCode struct {
+	Config ConfigKey
+	Rows   UsedSharedCodeRows
+}
+
 type Transformation struct {
-	Blocks []*Block
+	Blocks           []*Block
+	LinkToSharedCode *LinkToSharedCode
 }
```

Comment:
`sharedcode/links` - prepojenie transformacie a shared code bolo implementovane tak,
ze sa len mapovali hodnoty v `Config.Content` (pre kratkost casu pred release `v2.0`).

No bohuzial to nie je kompatibilne so sablonami, lebo neviem najst `ConfigId/RowId`, ktore mam nahradit.

Preto som pridal novu polozku `Transformation.LinkToSharedCode`, kde su IDs ulozene.